source/features/search-filters.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {XIcon} from '@primer/octicons-react';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import SearchQuery from '../github-helpers/search-query';
+import {getUsername} from '../github-helpers';
+
+function init(): void {
+	const items = [
+		['Forks', 'fork:true'],
+		['Private', 'is:private'],
+		['Yours', `user:${getUsername()}`]
+	].map(([name, filter]) => {
+		const item = <a className="filter-item" href={location.href}>{name}</a>;
+		// @ts-expect-error
```

Comment:
This should be fixed before merging. Let it fail or else we don't even see the error and help
```suggestion
```

Reply:
`item` is `JSX.Element` while `SearchQuery` expects `HTMLAnchorElement`.